modules/serverless-pingsource-yaml.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-pingsource.adoc
+
+The following sections describe how to create, verify and remove a basic PingSource using YAML files.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+
+.Procedure
+
+. To verify that `PingSource` is working, create a simple Knative
+service that dumps incoming messages to its log.
+
+. Copy the example YAML into a file named `service.yaml`:
++
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
++
+. Create the service by entering the command:
++
+----
+$ oc apply --filename service.yaml
+----
+
+. For each set of ping events that you want to request, create a PingSource in the same namespace as the event consumer.
++
+Copy the example YAML into a file named `ping-source.yaml`:
++
+----
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/2 * * * *"
+  jsonData: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
++
+Create the PingSource by entering the command:
++
+----
+$ oc apply --filename ping-source.yaml
+----
+
+. Check that the controller is mapped correctly by entering the following command and inspecting the output:
++
+----
+$ oc get pingsource.sources.knative.dev test-ping-source -oyaml
+----
++
+The output should be similar to:
++
+----
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  annotations:
+    sources.knative.dev/creator: minikube-user
```

Comment:
I'm not entirely sure, but it seems to me that we should get rid of minikube annotations in OpenShift documentation. :)

Reply:
@dsimansk I'd say you're right - can you please provide details for replacements for this tomorrow alongside the apiserversource examples? Thanks.